pkg/collector/tcp.go

```diff
@@ -59,7 +59,7 @@ func (cp *collectingProcess) handleTCPClient(conn net.Conn, wg *sync.WaitGroup)
 				}
 				break out
 			}
-			klog.Infof("Receiving %d bytes from %s", size, cp.address.String())
+			klog.Infof("Receiving %d bytes from %s", size, address)
```

Comment:
I guess this is a repeated log. Will log-level 2 be good here?

Reply:
Agreed.